ApollosProject/apollos-apps
# feat: Liking comments

## DESCRIPTION

### What does this PR do, or why is it needed?
Adds a user-likes table and mutations for liking or unliking a comment. Includes a new likedCount field on Comments.

Also includes the migrations for user-flags which were omitted before.

### How do I test this PR?
Create a comment:
```
mutation addComment {
addComment(parentId:"WeekendContentItem:f8460101368b38c5aa9bbea8b2b8a625", 
  text:"Look at me, a comment!", visibility:PUBLIC){
    id
  	isLiked
  	likedCount
  } 
}
```

Like it:
```
mutation likeComment {
  likeComment(commentId:"Comment:b364f263-862b-46d2-ac56-a85591dcb34d") {
    id
    isLiked
    likedCount
  }
}
```

Unlike it:
```
mutation unlikeComment {
  unlikeComment(commentId:"Comment:b364f263-862b-46d2-ac56-a85591dcb34d") {
    id
    isLiked
    likedCount
  }
}
```

Liking should increase its like count, unliking decrease. `isLiked` should only be true if you have liked this comment.

To really test the counting, try liking a comment, then editing the personId on that like to something else (to simulate as if another user had liked it), then send the `likeComment` migration again. The count should increase to 2.
